controller/src/pid.cpp

```diff
@@ -139,18 +139,18 @@ void pid_execute(const VentParams &params, SensorReadings *readings) {
   // Convert [kPa] to [cm H2O] in place
   readings->pressure_cm_h2o =
       get_pressure_reading(PressureSensors::PATIENT_PIN) * 10.1974f;
+
   // TODO(anyone): This value is to be the integration over time of the below
   // flow_ml_per_min. That is, you take the value calculated for flow_ml_per_min
   // and integrate it over time to get total volume at the current time. Don't
   // think this necessarily has to be done on the controller though?
   readings->volume_ml = 0;
+
   // Flow rate is inhalation flow minus exhalation flow. Positive value is flow
   // into lungs, and negative is flow out of lungs.
-  // Convert [meters^3/s] to [mL/min] in place
-  readings->flow_ml_per_min =
-      (pressure_delta_to_volumetric_flow(
-           get_pressure_reading(PressureSensors::INHALATION_PIN)) -
-       pressure_delta_to_volumetric_flow(
-           get_pressure_reading(PressureSensors::EXHALATION_PIN))) *
-      1000.0f * 1000.0f * 60.0f;
+  float flow = // [m^3/s]
+      pressure_delta_to_flow(
+          get_pressure_reading(PressureSensors::INHALATION_PIN) -
+          get_pressure_reading(PressureSensors::EXHALATION_PIN));
+  readings->flow_ml_per_min = flow * 1000.0f * 1000.0f * 60.0f;
```

Comment:
This is cleaner, looks good.

Reply:
Just to check -- this *is* correct, right?  Like, I think I'm fixing a bug, just want to be sure I'm not introducing one.

Reply:
Oh dear, this is actually incorrect. Good call. f(x-y) =/= f(x) - f(y) in this case. Intuitively it *makes* sense for some reason for this setup, but mathematically it's wrong. 

If you look at pressure_delta_to_volumetric, the equation that translates delta pressure to Q is essentially f(deltaP) = constant * sqrt(deltaP). The square root makes f(x-y) =/= f(x) - f(y).

Reply:
> f(x-y) =/= f(x) - f(y) in this case.

Agreed.

> The original is correct, this is not.

The function is called "pressure_delta_to_volumetric_flow" but when we do `f(x) - f(y)` aren't we passing in absolute pressures `x` and `y`?  Whereas `f(x-y)` is a pressure delta?